code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -204,21 +204,19 @@
 	if(href_list["operation"])
 		switch(href_list["operation"])	//toggles customizable behavioural protocols
 			if("authweapon")
-				auth_weapons = !auth_weapons
-			if("checkrecords")
-				check_records = !check_records
-			if("shootcrooks")
-				criminals = !criminals
+				TOGGLE_BITFIELD(turret_flags, TURRET_FLAG_AUTH_WEAPONS)

```

Comment:
these ugly ass defines still exist? smh

Reply:
Should I switch the cases where I've used it to `turret_flags ^= FLAG` instead?
Did a quick check and there is only one other instance of TOGGLE_BITFIELD being used in the entire game as well.

Reply:
Yeah, I'm removing those dumb defines in a PR